### PR TITLE
Use consistent CI tooling.  Include recent `local` single-repo feature.

### PR DESCRIPTION
**Pull request by joewagner**
## Overview
This PR brings this repo up to date with the recent singular package repo feature additions, and also includes changes to linting and testing. 

## Details
The linting and testing changes are significant because each of the packages had slight variations on how that tooling was setup.  The majority of the changes were bringing `local` and `cli` up to date with `sdk`

### CODE REVIEW

**Comment by joewagner** [comment]
@carsonfarmer There's a ton of changes here, as such it's potentially better to just look at the state of the branch then look at the diff.
Everything works as expected for me locally, but unfortunately the test check just hangs forever.  I'm going to try to setup a local github action runner and see if I can gain some insight into why the tests are hanging.
If you get a free cycle, it would be great to get feedback on how all of this is setup.

**Comment by joewagner** [comment]
UPDATE: 
I had to remove the SDK's playwright browser tests, but all tests are now passing (and even running in parallel).  I'll take this out of draft aand if everything looks good we can start using this repo for releases.

File: packages/sdk/src/helpers/config.ts [hunk 1]
```diff
@@ -69,3 +78,23 @@ export async function extractChainId(conn: Config = {}): Promise<number> {
...
+  if (!fs.existsSync(filepath)) {
+    fs.writeFileSync(filepath, JSON.stringify({}));
+  }
+
+  return fs.readFileSync(filepath);
+};
+
+export function jsonFileAliases(filepath: string): AliasesNameMap {
+  return {
+    read: async function (): Promise<NameMapping> {
```
Comment (joewagner) [inline packages/sdk/src/helpers/config.ts:92]: from https://github.com/tablelandnetwork/js-tableland/pull/542

**Comment by joewagner** [comment]
@asutula I tagged you as a reviewer here.  @carsonfarmer probably has the most context to do a review, but if this kind of setup works well I'd suggest that we setup studio in a similar way.  Specifically, the app could be a package and the cli could be a package.

**Comment by joewagner** [comment]
@carsonfarmer This PR is kind of a lot of changes, is there anything I can do to make it easier to review?

File: packages/sdk/src/database.ts [hunk 2]
```diff
@@ -228,9 +235,31 @@ async function normalizedToRunnables(
...
   if (normalized.tables.length > 1) {
-    throw new Error(
-      "each statement can only touch one table. try batching statements based on the table they mutate."
-    );
+    let isMutatingMultiple = true;
+    for (const stmt of normalized.statements) {
+      // re-normalize so we can be sure we've isolated each statement and its tableId
+      const norm = await normalize(stmt);
+
+      if (norm.tables.length > 1) {
```
[the remark this answers is not in the document]
Comment (carsonfarmer) [reply packages/sdk/src/database.ts:243]: I know this is from a previous PR, but we might want to explore handling more of this in the parser?

**Review by carsonfarmer — APPROVED** [review]
Ok, wow. This was a lot of work @joewagner! Really great stuff. Let's get this merged in, and we'll move forward from a unified point.

Really great work here, just a few tiny nits. We'll likely fine a few issues once we get this merged as well... so we can create separate issues for this as they crop up.

File: packages/sdk/src/database.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (joewagner) [reply packages/sdk/src/database.ts:243]: Good call!

**Pull request merged**